source/models/multi_signing_address.cpp

```diff
@@ -0,0 +1,86 @@
+//
+// MIT License
+//
+// Copyright (c) 2017-2018 Thibault Martinez and Simon Ninon
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+//
+//
+
+#include <iota/constants.hpp>
+#include <iota/errors/illegal_state.hpp>
+#include <iota/models/multi_signing_address.hpp>
+
+namespace IOTA {
+
+namespace Models {
+
+MultiSigningAddress::MultiSigningAddress(const Types::Trytes& address, const int64_t& balance,
+                                         const int32_t& keyIndex, const int32_t& security) {
+  setAddress(address);
```

Comment:
is there some advantages of calling the default ctor of Address instead of the one you override?
This code is a bit duplicated and could be removed by calling the appropriate constructor.
Or is it related to some kind of optimization to avoid calling some virtual methods?